src/classes/HH_INamingImpl.cls

```diff
@@ -87,35 +87,37 @@ public with sharing class HH_INamingImpl implements HH_INaming {
     * @description To allow the NPSP code to know which fields to query for on Contact, before handing them off to the naming routines.
     * @return set of strings of all fields referenced from Contact.
     */
-    private static Set<String> setHouseholdNameFieldsOnContact;
-    public set<string> setHouseholdNameFieldsOnContact() {
-        if (setHouseholdNameFieldsOnContact == null) {
-            set<string> setStrField = new set<string>();
+    private static Set<String> householdNamingFields;
+    public Set<String> householdNamingFields() {
+        if (householdNamingFields == null) {
+            Set<String> setStrField = new Set<String>();
             NameSpec ns;
             ns = new NameSpec(hns, 'Household_Name_Format__c');
             setStrField.addAll(ns.setStrFieldsInNameSpec());
             ns = new NameSpec(hns, 'Formal_Greeting_Format__c');
             setStrField.addAll(ns.setStrFieldsInNameSpec());
             ns = new NameSpec(hns, 'Informal_Greeting_Format__c');
             setStrField.addAll(ns.setStrFieldsInNameSpec());
-            setHouseholdNameFieldsOnContact = setStrField;
+            householdNamingFields = setStrField;
         }
-        return setHouseholdNameFieldsOnContact;
+        return householdNamingFields;
     }
 
     /*******************************************************************************************************
     * @description our cached copy of the Household Naming Settings
     */
     private Household_Naming_Settings__c hns {
     	get {
-    		if (hns == null) hns = UTIL_CustomSettingsFacade.getHouseholdNamingSettings();
+    		if (hns == null) {
```

Comment:
Indent is a little off here (mainly I think it's a [tab] instead of spaces).

Reply:
I auto aligned it.  Let me know if it still looks off.